migrations/20240217102522-create-settings.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable(
+      'Settings',
+      {
+        id: {
+          allowNull: false,
+          autoIncrement: true,
+          primaryKey: true,
+          type: Sequelize.INTEGER,
+        },
+        name: {
+          allowNull: false,
+          type: Sequelize.STRING,
+        },
+        data_type: {
+          allowNull: false,
+          type: Sequelize.STRING,
```

Comment:
This should be an enum of allowed values instead of strings

Reply:
[Done](https://github.com/ananth243/nest-starter-project/blob/732f3cf7c7bc7cf884500fc5eda0ff5aa6709c60/migrations/20240217102522-create-settings.js#L21)